### PR TITLE
Pttb 290 token should be shown in response

### PR DESCRIPTION
**Pull request by Amogh-svge**
reponse should contain token only in local and development server it should not contain token in production serve

### CODE REVIEW

File: app/Http/Controllers/Actions/Invite/InviteOthersAction.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     * @param MemberInviteRequest $request
+     * @param InviteService $inviteService
+     * @throws QueryException
+     * @throws Exception
+     * @return JsonResponse
+     */
+
```
Comment (smellycat5) [inline app/Http/Controllers/Actions/Invite/InviteOthersAction.php:43]: why is there extra space here. plseae remove this

File: app/Http/Controllers/Actions/Invite/InviteOthersAction.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    use HttpResponses;
+
+    protected InviteService $inviteService;
+
+    /**
+     *
+     * @param UserService $userService
+     * @param InviteService $inviteService
+     */
+
```
Comment (smellycat5) [inline app/Http/Controllers/Actions/Invite/InviteOthersAction.php:26]: remove extra space here

File: app/Http/Controllers/AdminController.php [hunk 3]
```diff
@@ -22,16 +22,20 @@ class AdminController extends Controller
...
-    private UserService $userService;
+    protected UserService $userService;
+    protected InviteService $inviteService;
+
 
     /**
      *
      * @param UserService $userService
+     * @param InviteService $inviteService
      */
```
Comment (smellycat5) [inline app/Http/Controllers/AdminController.php:34]: unnecssary blank space

File: app/Services/InviteService.php [hunk 4]
```diff
@@ -6,38 +6,62 @@
...
 
 class InviteService
 {
-    public function generateToken(): string
+    protected InviteToken $inviteToken;
+
+    /**
+     * @param InviteToken $inviteToken
+     */
+
```
Comment (smellycat5) [inline app/Services/InviteService.php:23]: same here remove it lol

File: app/Services/InviteService.php [hunk 5]
```diff
@@ -6,38 +6,62 @@
...
+        $random = Str::random(45);
+        return $random . uniqid();
     }
 
+    /**
+     *
+     * @param array $credentials
+     * @return void
+     */
+
```
Comment (smellycat5) [inline app/Services/InviteService.php:44]: same here

**Review by smellycat5 — COMMENTED** [review]
remove extra spaces

File: app/Http/Controllers/Actions/Invite/InviteOthersAction.php [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    public function __construct(InviteService $inviteService)
+    {
+        $this->inviteService = $inviteService;
+    }
+
+    /**
+     * Handle the incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
```
[the remark this answers is not in the document]
Comment (Amogh-svge) [reply app/Http/Controllers/Actions/Invite/InviteOthersAction.php:35]: ahh solved

**Review by sahajmalla — CHANGES_REQUESTED** [review]
Why should not response content token in the production env and only in the local and dev servers?

**Pull request closed without merging**